sdk/storage/azblob/CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 * Updated to latest `azcore`.  Public surface area is unchanged.
 
 ### Bugs Fixed
```

Comment:
Is there a plan to add all the missing changelog entries from when the storage dev branch was merged to main?

Reply:
Yes, I have plans to add more changelogs. There is one more bug which I am planning to solve after which I'll do the next azblob release.